test/parallel/test-child-process-exec-maxBuffer.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/1901
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+const unicode = '中文测试'; // Length = 4, Byte length = 13
+
+if (process.argv[2] === 'stdout') {
+  console.log(unicode);
+  return;
+}
+
+if (process.argv[2] === 'stderr') {
+  console.error(unicode);
+  return;
+}
+
+const cmd = `${process.execPath} ${__filename}`;
```

Comment:
Argh, good point, of course. And while it's fixable, it probably gets pretty convoluted quickly checking for quotation marks, `$`, and other things. Guess I'll switch to using `echo` but I was hoping to avoid echo`+unicode weirdness on Windows and possibly elsewhere.